source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py

```diff
@@ -332,13 +332,13 @@ class RidesharingInformationSerializer(PbNestedSerializer):
     seats = SeatsDescriptionSerializer(display_none=False)
 
 
-class OdtInformationSerializer(PbNestedSerializer):
+class   BookingRuleSerializer(PbNestedSerializer):
```

Comment:
```suggestion
class BookingRuleSerializer(PbNestedSerializer):
```
Petits espaces en trop